python/src/OptimizationResult_doc.i.in

```diff
@@ -371,11 +371,11 @@ status : int
 // ---------------------------------------------------------------------
 
 %feature("docstring") OT::OptimizationResult::getStatus
-"Accessor to the status string.
+"Accessor to the generic status.
 
 Returns
 -------
-status : str
+status : int
     Status flag, possible values SUCCEEDED, FAILURE"
```

Comment:
During our discussion at the last technical committee meeting, the use of generic status flags was called into question because it is not object-oriented. That said, I understand this PR only documents this functionality which was implemented before, so this comment does not impact the current PR but is something to think about in the future.